react-redux-app/src/componets/FormEdit/FormEdit.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import bucet from "../../img/bucet.svg";
+import { updateTheme, toggleEdit } from "../../store/listSlice";
+import "./FormEdit.css";
+import { useDispatch, useSelector } from "react-redux";
+import Modal from "../Modal/Modal";
+import * as yup from "yup";
+import { useFormik } from "formik";
+
+const FormEdit = () => {
+  const [modalActive, setModalActive] = useState(false);
+  const allTheme = useSelector((state) => state.themes.lists);
+  const editMode = useSelector((state) => state.themes.editMode);
+  const findId = useSelector((state) => state.themes.editThemeId);
+  const dispatch = useDispatch();
+  const toggleEditeForm = () => dispatch(toggleEdit());
+  const editTheme = allTheme.find((el) => el.id === findId);
+  const onSubmit = (values) => {
+    dispatch(updateTheme({ ...values }));
+    toggleEditeForm();
+  };
+
+  const validationSchema = yup.object({
```

Comment:
you created already validationSchema in Form.js. take out it pls in separate folder 'helpers', it must be in src